lib/pure/json.nim

```diff
@@ -519,9 +519,9 @@ proc `{}`*(node: JsonNode, keys: varargs[string]): JsonNode =
   ## This proc can be used to create tree structures on the
   ## fly (sometimes called `autovivification`:idx:):
   ##
-  ## .. code-block:: nim
-  ##   myjson{"parent", "child", "grandchild"} = newJInt(1)
-  ##
+  runnableExamples:
+    doAssert myjson{"parent", "child", "grandchild"} == newJInt(1)
```

Comment:
this one is obviously wrong since myjson isn't defined here

Reply:
done